src/services/api.js

```diff
@@ -0,0 +1,10 @@
+export default {
+    getNews() {
+        return fetch('https://newsapi.org/v2/everything?q=bitcoin&from=2018-11-04&sortBy=publishedAt&apiKey=382852b6e4974fde8afeff04975c74b1')
```

Comment:
```suggestion
        return fetch('https://newsapi.org/v2/everything?q=bitcoin&sortBy=publishedAt&apiKey=382852b6e4974fde8afeff04975c74b1')
```
For whatever reason, it looks like this api has restrictions on how far back you can search when you are on the free plan. When I remove the time parameter it showed results. Api's can be tricky that way!